### PR TITLE
Refactor folder picker to be identical to the one used in PM web app

### PR DESCRIPTION
**Pull request by fdaveine**
-

### CODE REVIEW

**Comment by antoinelibert** [comment]
I have some warnings coming up when navigating folders tree. Is that normal?

```
16:05:20.986 › last API request fail with 403 status, accessToken is not valid
16:05:20.992 › try to replay last request with refreshed token
```

**Comment by fdaveine** [comment]
> I have some warnings coming up when navigating folders tree. Is that normal?
> 
> ```
> 16:05:20.986 › last API request fail with 403 status, accessToken is not valid
> 16:05:20.992 › try to replay last request with refreshed token
> ```
> 
> Edit: it's on prod.

No it isn't by I already seen this a while ago too, it look like the access token date is always considered as out of date or something like that, I will see what going on.

**Comment by antoinelibert** [comment]
> it looks like the access token date is always considered as out of date

Timezone issue?

**Comment by fdaveine** [comment]
> > it looks like the access token date is always considered as out of date
> 
> Timezone issue?

Will be fixed in https://github.com/exotic-matter-sas/paper-matter-import-export/issues/12

File: src/renderer/components/HomePage/FolderPickerModal/FTLTreeItem.vue [hunk 1]
```diff
@@ -5,29 +5,34 @@
...
-        <font-awesome-icon :icon="isOpen || item.id==null ? 'folder-open' : 'folder'"/>
+    <b-link class="expand-folder-child" v-if="item.has_descendant && !item.is_root" @click="toggle">
+      <b-spinner v-if="loading" small></b-spinner>
+      <font-awesome-icon v-else :icon="isOpen ? ['far', 'minus-square'] : ['far', 'plus-square']"/>
+    </b-link>
+    <b-link @click.prevent="folderSelected" @dblclick.prevent="toggle" class="d-block"
+            :class="{'font-weight-bold': item.has_descendant, selected: unSavedImportDestination && unSavedImportDestination.id === item.id}">
+      <span class="target-folder-name mx-2" :title="item.name">
+        <font-awesome-icon :icon="isOpen || item.is_root ? 'folder-open' : 'folder'"/>
         &nbsp;{{ item.name }}&nbsp;
```
Comment (antoinelibert) [inline src/renderer/components/HomePage/FolderPickerModal/FTLTreeItem.vue]: Is the `&nbsp;` expected?

Comment (fdaveine) [reply src/renderer/components/HomePage/FolderPickerModal/FTLTreeItem.vue]: de362fe

**Pull request merged**